<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-30T18:00:20 IST" name="Suite" finished-at="2025-08-30T18:00:41 IST" duration-ms="21434">
    <groups>
    </groups>
    <test started-at="2025-08-30T18:00:20 IST" name="Test" finished-at="2025-08-30T18:00:41 IST" duration-ms="21434">
      <class name="com.testcase.DemoKartTestSuite">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:20 IST" name="setUp" finished-at="2025-08-30T18:00:34 IST" duration-ms="14463" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="initializeSoftAssert()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:34 IST" name="initializeSoftAssert" finished-at="2025-08-30T18:00:34 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeSoftAssert -->
        <test-method signature="tc001_addToCartWithoutLogin()[pri:1, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:34 IST" name="tc001_addToCartWithoutLogin" finished-at="2025-08-30T18:00:35 IST" duration-ms="162" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc001_addToCartWithoutLogin -->
        <test-method signature="tc002_loginWithValidCredentials()[pri:2, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:35 IST" name="tc002_loginWithValidCredentials" finished-at="2025-08-30T18:00:35 IST" depends-on-methods="com.testcase.DemoKartTestSuite.tc001_addToCartWithoutLogin" duration-ms="297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc002_loginWithValidCredentials -->
        <test-method is-config="true" signature="initializeSoftAssert()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:35 IST" name="initializeSoftAssert" finished-at="2025-08-30T18:00:35 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeSoftAssert -->
        <test-method signature="tc003_welcomeBackToastOnProductPage()[pri:3, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:35 IST" name="tc003_welcomeBackToastOnProductPage" finished-at="2025-08-30T18:00:36 IST" depends-on-methods="com.testcase.DemoKartTestSuite.tc002_loginWithValidCredentials" duration-ms="1116" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc003_welcomeBackToastOnProductPage -->
        <test-method is-config="true" signature="initializeSoftAssert()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:35 IST" name="initializeSoftAssert" finished-at="2025-08-30T18:00:35 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeSoftAssert -->
        <test-method is-config="true" signature="initializeSoftAssert()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:36 IST" name="initializeSoftAssert" finished-at="2025-08-30T18:00:36 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeSoftAssert -->
        <test-method signature="tc004_addToCartAndValidateUrl()[pri:4, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:36 IST" name="tc004_addToCartAndValidateUrl" finished-at="2025-08-30T18:00:37 IST" depends-on-methods="com.testcase.DemoKartTestSuite.tc003_welcomeBackToastOnProductPage" duration-ms="453" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Phone 15 Pro Max has been added to your cart] but found [iPhone 15 Pro Max has been added to your cart]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Phone 15 Pro Max has been added to your cart] but found [iPhone 15 Pro Max has been added to your cart]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.testcase.DemoKartTestSuite.tc004_addToCartAndValidateUrl(DemoKartTestSuite.java:69)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc004_addToCartAndValidateUrl -->
        <test-method signature="tc005_checkoutAndOrderConfirmation()[pri:5, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:37 IST" name="tc005_checkoutAndOrderConfirmation" finished-at="2025-08-30T18:00:41 IST" duration-ms="4284" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Order placed successfully!] but found [Payment failed]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Order placed successfully!] but found [Payment failed]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.testcase.DemoKartTestSuite.tc005_checkoutAndOrderConfirmation(DemoKartTestSuite.java:80)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc005_checkoutAndOrderConfirmation -->
        <test-method is-config="true" signature="initializeSoftAssert()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:37 IST" name="initializeSoftAssert" finished-at="2025-08-30T18:00:37 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeSoftAssert -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:com.testcase.DemoKartTestSuite@29f7cefd]" started-at="2025-08-30T18:00:41 IST" name="afterClass" finished-at="2025-08-30T18:00:41 IST" duration-ms="320" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.testcase.DemoKartTestSuite -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
